### ISSUE TITLE
NetworkPolicy ipv6 doesn't work. 

### ISSUE THREAD

**Issue by xujunjie-cover** [cloudnativelabs/kube-router#1653]
**What happened?**
NetworkPolicy ipv6 doesn't work. ipset name and ip6tables rule are not equal

**What did you expect to happen?**
networkpolicy works with ipv6

**How can we reproduce the behavior you experienced?**
1. create test pod
2. create networkpolicy
```
apiVersion: networking.k8s.io/v1
kind: NetworkPolicy
metadata:
  name: test-network-policy
  namespace: default
spec:
  podSelector:
    matchLabels:
      app: nginx
  policyTypes:
  - Ingress
  ingress:
  - from:
    - podSelector:
        matchLabels:
          app: test
    ports:
      - protocol: TCP
        port: 80
```

**Screenshots / Architecture Diagrams / Network Topologies **
If applicable, add those here to help explain your problem.

** System Information (please complete the following information):**
 - Kube-Router Version (`kube-router --version`): v2.1.0
 - Kube-Router Parameters:         - --run-router=false  - --run-firewall=true - --run-service-proxy=false - --bgp-graceful-restart=true - --enable-ipv6=true
 - Kubernetes Version (`kubectl version`) : 1.24.17
 - Cloud Type: on premise
 - Kubernetes Deployment Type: Kubeadm
 - Kube-Router Deployment Type: DaemonSet
 - Cluster Size: 2

** Logs, other output, metrics **
```
 Aborting sync. Failed to run iptables-restore: failed to call ip6tables-restore: exit status 2 (ip6tables-restore v1.8.7 (legacy): Set KUBE-SRC-YZGP7RC3Z53QX6UT doesn't exist.
```

logs report ipset KUBE-SRC-YZGP7RC3Z53QX6UT not exist, but the inet6:KUBE-SRC-YZGP7RC3Z53QX6UT is correct. the ipset used by ip6tables without inet6 Prefix.

https://github.com/cloudnativelabs/kube-router/blob/b423b1feb191467db51eed3edd4e6a988d937ff4/pkg/controllers/netpol/policy.go#L215-L233
L224 create ipset name srcPodIPSetName with preifx "inet6"
L242 appendRuleToPolicyChain only use srcPodIPSetName

how to solve?  add prefix in appendRuleToPolicyChain ?

**Comment by aauren** [comment]
Thanks @xujunjie-cover!

This is a great find and I'm sad that we missed this in our testing. :disappointed: 

I'll check out your PR!

**Comment by aauren** [comment]
Fixed via #1654
